doc/source/whatsnew/v0.24.0.rst

```diff
@@ -1261,7 +1261,7 @@ Numeric
 Strings
 ^^^^^^^
 
--
+- BUG in :meth:`Index.str.partition` was not nan-safe (:issue:`23558`).
```

Comment:
can you following existing style BUG -> Bug

Reply:
followed the existing style, now
Whoops. Good catch, thanks. Fixed in 25d5743